internal/rainbowprint/colors.go

```diff
@@ -0,0 +1,127 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+   Copyright (c) 2023, Unikraft GmbH and The KraftKit Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package rainbowprint
```

Comment:
Can you please add a [package description](https://go.dev/blog/godoc) right above this?